sound/soc/codecs/rt711-sdca-sdw.c

```diff
@@ -142,21 +138,6 @@ static int rt711_sdca_update_status(struct sdw_slave *slave,
 	if (status == SDW_SLAVE_UNATTACHED)
 		rt711->hw_init = false;
 
-	if (status == SDW_SLAVE_ATTACHED) {
-		if (rt711->hs_jack) {
-			regcache_cache_only(rt711->regmap, false);
-			regcache_cache_bypass(rt711->regmap, true);
-			regmap_write(rt711->regmap, SDW_SCP_SDCA_INTMASK1,
-				SDW_SCP_SDCA_INTMASK_SDCA_0);
-			regmap_write(rt711->regmap, SDW_SCP_SDCA_INTMASK2,
-				SDW_SCP_SDCA_INTMASK_SDCA_8);
```

Comment:
@plbossart If we remove the SDW_SCP_SDCA_INTMASKX register operations here, we need to add it somewhere. All interrupt mask bits will become 0 after any reset.

Reply:
@bardliao it's already done in the hw init part. There is no reason to do this twice.